package/lib/src/modal/swipe_dismiss_config.dart

```diff
@@ -0,0 +1,33 @@
+/// Configuration class for handling swipe-to-dismiss gestures.
+///
+/// The [SwipeDismissConfig] class provides parameters to control the behavior
+/// of swipe-to-dismiss interactions for modals. It defines the minimum velocity
+/// required for the swipe gesture ([minFlingVelocity]) and the minimum distance
+/// the user needs to drag ([minDragDistance]) for a swipe to result in a dismissal.
+class SwipeDismissConfig {
```

Comment:
I prefer `Sensitivity` over `Sensitivity` as I feel the latter term is a bit too abstract...

Reply:
Sorry, that was a typo. I meant that the name `SwipeDismissSensitivity` might be preferable to `SwipeDismissConfig`, as the word `Config` is a rather broad term.